cmd/commander/commander.go

```diff
@@ -63,13 +63,17 @@ func createTestCommand() cli.Command {
 				Name:  "dir",
 				Usage: "Execute all test files in a directory sorted by file name, this is not recursive - e.g. /path/to/test_files/",
 			},
+			cli.StringFlag{
+				Name:  "filter",
+				Usage: "Filter tests by a given regex pattern. Tests are filtered by its title.",
+			},
 		},
 		Action: func(c *cli.Context) error {
 			if c.Bool("verbose") {
```

Comment:
Just a comment, this can be a removed. We do it in `Test_Command.go`. I'll add it in #126, unless we have a reason not too